lib/adapters/MochaAdapter.js

```diff
@@ -1,16 +1,19 @@
 import EventEmitter from 'events'
-import { Test, Suite } from '../Data.js'
+import {Test, Suite} from '../Data.js'
 
 export default class MochaAdapter extends EventEmitter {
   constructor (mocha) {
     super()
 
     this.mocha = mocha
+
     mocha.reporter((runner) => {
       this.runner = runner
+
       runner.on('start', this.onStart.bind(this))
       runner.on('suite', this.onSuite.bind(this))
       runner.on('test', this.onTest.bind(this))
+      runner.on('pending', this.onPending.bind(this))
```

Comment:
That's a category of change I expect to be covered by tests. Since you didn't modify any tests, there must be missing coverage, right?